src/validators.js

```diff
@@ -51,6 +51,7 @@ const extractParams = (req, res, next) => {
 		if (!req['recordId'] && pathParams[2]) {
 			req['recordId'] = isHexString.test(pathParams[2]) ? pathParams[2] : undefined;
 		}
+		console.log(req.headers)
```

Comment:
Can we remove this log?

Reply:
Yes! Sorry about that I was testing and forgot it in there